web/styles/app_variables.css

```diff
@@ -123,9 +123,13 @@
     --message-box-avatar-width: 35px;
     --message-box-avatar-height: var(--message-box-avatar-width);
     --message-box-avatar-column-width: 46px;
-    /* 6px appears to provide 5px of space between the avatar
-       and the selected-message outline, which is offset by -1px. */
-    --message-box-vertical-margin: 6px;
+    /* Increase the margin here to account for the
+       focus ring, which is is offset by -1px, as well
+       as the vertical height between the hover icons
+       and focus ring. */
+    --message-box-vertical-margin: calc(
+        var(--markdown-interelement-space-px) * 1.4
+    );
```

Comment:
Would it make sense for this formula to involve a `x * y +1`, to account for the focus ring more precisely?

Reply:
I had that initially, but it seemed to leave the avatar sitting too high in the message box relative to the username and focus ring.

Reply:
I do think there's a more rigorously mathematical way to arrive at a value here, but I was just trying to get closer to what was being asked for in the [CZO discussion](https://chat.zulip.org/#narrow/stream/101-design/topic/spacing.20around.20message-box.20elements/near/1838926) relative to Vlad's designs